include/openssl/ssl.h

```diff
@@ -323,6 +323,9 @@ typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
         SSL_OP_NO_TLSv1|SSL_OP_NO_TLSv1_1|SSL_OP_NO_TLSv1_2)
 # define SSL_OP_NO_DTLS_MASK (SSL_OP_NO_DTLSv1|SSL_OP_NO_DTLSv1_2)
 
+/* Disallow all renegotiation */
+# define SSL_OP_NO_RENEGOTIATION                         0x40000000U
```

Comment:
The non-conflict with SSL_OP_NETSCAPE_DEMO_CIPHER_CHANGE_BUG from 1.0.2 should be a non-bug, if I understand correctly.

Reply:
Right. SSL_OP_NETSCAPE_DEMO_CIPHER_CHANGE_BUG was set to 0 in 1.1.0. Since we don't promise binary compat between 1.0.2 and 1.1.0 this should be fine.

Reply:
Right, the issue is that in 1.1.0, SSL_OP_ALL included unused bits in its value. So, if a program compiled with 1.1.0 uses it's value of SSL_OP_ALL against a dynamically loaded 1.1.1 or later, we can't have unexpected (i.e. new) options enabled. Not an issue cross major revs: 1.0.x -> 1.1.x.